tests/TestLaunchExporter.py

```diff
@@ -49,5 +69,26 @@ def test_without_params(self):
             parse_params()
         self.assertEqual(se.exception.code, 0, 'PORT or INVENTORY are not set properly in ENV or command line!')
 
+import collectors.CollectorUp
+import importlib
+from exporter import initialize_collector_by_name
+class TestCollectorInitialization(TestCase):
+    def test_valid_collector(self):
+        collector = initialize_collector_by_name('CollectorUp')
+        self.assertIsInstance(collector, collectors.CollectorUp.CollectorUp)
+
+    @patch('builtins.print')
+    def test_with_bogus_collector(self, mocked_print):
+        collector = initialize_collector_by_name('BogusCollector')
+        self.assertIsNone(collector)
+        self.assertEqual(mocked_print.mock_calls, [call('No Collector "BogusCollector" defined. Ignoring...')])
+
+    @patch('builtins.print')
+    def test_with_invalid_collector(self, mocked_print):
+        importlib.import_module = MagicMock(return_value=collectors.CollectorUp)
+        collector = initialize_collector_by_name('ClassNotDefinedCollector')
+        self.assertIsNone(collector)
+        self.assertEqual(mocked_print.mock_calls, [call('Unable to initialize "ClassNotDefinedCollector". Ignoring...')])
+
```

Comment:
Good idea, please move to dedicated test file. Maybe take another collector, not sure if we are keeping the collectorUp-collector.